app/assets/stylesheets/_mixins.scss

```diff
@@ -72,4 +72,13 @@
       $light-gray
     );
   }
-}
\ No newline at end of file
+}
+
+// TODO: comment this sucker
+@mixin font-src($font) {
```

Comment:
This mixin is required so that we can load a font URL via Storybook's webpack or Sprockets. The `$environment` variable is only ever set when webpack for Storybook is running.